randovania/games/dread/json_data/Burenia.txt

```diff
@@ -1832,29 +1855,41 @@ Extra - asset_id: collision_camera_015
   * Layers: default
   > Door to Main Hub Tower Middle (Charge)
       Trivial
-  > Event - Blob from nearby (db_reg_aq_012)
-      Shoot Beam
+  > Door to Main Hub Tower Middle (Plasma)
+      All of the following:
+          Wave Beam and Disabled Door Lock Randomizer
+          Any of the following:
+              Gravity Suit or Spider Magnet or Movement (Beginner) or Perform WBJ
```

Comment:
As above, this requires morph as you're doing a coyote jump out of the morph tunnel
```suggestion
          Gravity Suit or Spider Magnet or Perform WBJ
          Morph Ball and Movement (Beginner)
```